server/src/main/resources/db/V8__create_teams_table.sql

```diff
@@ -0,0 +1,14 @@
+DROP TABLE IF EXISTS team;
```

Comment:
You're going to need to rename this to have the next migration version number...looks like probably V10 if you act fast.

Reply:
renamed to V10